patches/@nozbe+watermelondb+0.23.0.patch

```diff
@@ -1,3 +1,29 @@
+diff --git a/node_modules/@nozbe/watermelondb/adapters/sqlite/encodeQuery/index.js b/node_modules/@nozbe/watermelondb/adapters/sqlite/encodeQuery/index.js
+index 4d49faf..262edb5 100644
+--- a/node_modules/@nozbe/watermelondb/adapters/sqlite/encodeQuery/index.js
++++ b/node_modules/@nozbe/watermelondb/adapters/sqlite/encodeQuery/index.js
+@@ -60,6 +60,10 @@ var encodeComparison = function (table, comparison) {
+     return right.values ? "between ".concat((0, _encodeValue.default)(right.values[0]), " and ").concat((0, _encodeValue.default)(right.values[1])) : '';
+   }
+ 
++  if (comparison.operator === 'like' && !!comparison.right.value && typeof comparison.right.value === 'string') {
```

Comment:
On the PR description you said you dropped the WMDB idea in favor of the slugify one.
Yet there's this change.
What does it do?